core/domain/prod_validation_jobs_one_off.py

```diff
@@ -40,13 +42,53 @@ def _external_id_relationships(cls):
             model to validate, and whose values are the corresponding model
             classes.
         """
-        raise NotImplementedError
+        return {}
```

Comment:
This should still be NotImplementedError (for the base class), to guard against cases where this isn't implemented in a subclass, I think.